server/migrations/1699048909312_add-withdrawn-date-to-participant-status-infos-view.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = async (pgm) => {
+  await pgm.sql(`
+    CREATE OR REPLACE VIEW public.participants_status_infos AS
+    SELECT p.id,
+      p.body,
+      p.search,
+      p.created_at,
+      p.updated_at,
+      jsonb_agg(ps.*) FILTER (WHERE ps.id IS NOT NULL) AS status_infos,
+      jsonb_agg(DISTINCT jsonb_build_object('id', rosstatus.id, 'data', rosstatus.data, 'status', rosstatus.status, 'site_id', rosstatus.site_id, 'rosSite', ros_site.*)) FILTER (WHERE rosstatus.id IS NOT NULL) AS ros_infos,
+      case when p.body->>'interested' = 'withdrawn'
+        then (
+          select
+          h.timestamp
+          from jsonb_to_recordset(p.body->'history') as h(timestamp timestamp, changes jsonb )
+          where (select * from jsonb_array_elements(h.changes) limit 1)->>'to' = 'withdrawn'
```

Comment:
`h.changes[0]` -> `(select * from jsonb_array_elements(h.changes) limit 1)`
v11 doesn't support accessing jsonb of array by index. that's what the error, 'cannot subscript type jsonb because it is not an array' meant.